docs/core/extensions/snippets/configuration/console-basic-builder/console-basic-builder.csproj

```diff
@@ -2,14 +2,14 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
```

Comment:
This project is not part of the solution. Is it on purpose?

Reply:
Yes, there are several projects that aren't necessarily part of a solution. I think that's fine.